.travis.yml

```diff
@@ -25,3 +25,6 @@ matrix:
   - name: "Lint and Type Checking"
     script:
     - docker run --rm --network=host --entrypoint="" azafea-tests pipenv run lint
+  - name: Update ReadTheDocs
+    script:
+    - docker run --rm azafea-tests deploy-documentation --token "$RTD_TOKEN"
```

Comment:
Is it correct to run this on pull requests, as is currently done? e.g. https://travis-ci.org/github/endlessm/azafea/jobs/746713046

Reply:
I think we can merge this one. There isn't anything to deploy (except for the token that's already in place).

Reply:
What happens when this trigger runs on a pull request? Does the PR's docs get deployed to RTD?

Reply:
I now understand what you meant in the previous comment. Yes, it would run but it would re-deploy the `master` branch.